src/components/DayOverview/DayOverviewPdf.tsx

```diff
@@ -79,16 +79,19 @@ const dateTimeToAMPM = (datetime: string): string => {
     return new Date(datetime).getHours() <= 11 ? "AM" : "PM";
 };
 
-const CustomSVG: React.FC<CustomSVGProps> = ({ icon, color, fill }) => (
-    <Svg width={11} height={11} viewBox="0 0 512 512" style={styles.svg}>
-        <Path
-            fill={fill ? color : undefined}
-            stroke={color}
-            strokeWidth={40}
-            d={typeof icon.icon[4] === "string" ? icon.icon[4] : icon.icon[4][0]}
-        />
-    </Svg>
-);
+const CustomSVG: React.FC<CustomSVGProps> = ({ icon, color, fill }) => {
+    const path = icon.icon[4];
```

Comment:
not sure what `path` refers to, can you think of a better name?

Reply:
See if `svgPath` is better because the literal meaning of those string is just SVG path